packages/opentelemetry-sdk-trace-base/src/BasicTracerProvider.ts

```diff
@@ -85,12 +85,19 @@ export class BasicTracerProvider implements TracerProvider {
       resource: this.resource,
     });
 
-    const defaultExporter = this._buildExporterFromEnv();
-    if (defaultExporter !== undefined) {
-      const batchProcessor = new BatchSpanProcessor(defaultExporter);
-      this.activeSpanProcessor = batchProcessor;
+    if (config.spanProcessors?.length) {
+      this._registeredSpanProcessors = [...config.spanProcessors];
+      this.activeSpanProcessor = new MultiSpanProcessor(
+        this._registeredSpanProcessors
```

Comment:
I think  we could actually get rid of `_registeredSpanProcessors` now, right? 🙂

Reply:
There is lot of tests that make use of that property to do some assertions. Like https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts#L260-L268